config/webpack.plugins.js

```diff
@@ -11,7 +11,6 @@ const plugins = [
   new WriteFileWebpackPlugin(),
   new webpack.SourceMapDevToolPlugin({
     test: /\.js/i,
-    exclude: /node_modules/i,
```

Comment:
Do we want to include node_modules as well? Sounds reasonable, just wanted to confirm.

Reply:
Yes. That is actually what fixes the problem. The warnings are about node modules source mapping.